standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/txn/TxnStore.java

```diff
@@ -115,6 +115,7 @@ enum MUTEX_KEY {
   // Compactor types
   char MAJOR_TYPE = 'a';
   char MINOR_TYPE = 'i';
+  char REBALANCE_TYPE = 'r';
```

Comment:
would be great to have an enum for that

Reply:
We are using these consts only in 
```
org.apache.hadoop.hive.metastore.txn.TxnUtils#thriftCompactionType2DbType
org.apache.hadoop.hive.metastore.txn.TxnUtils#dbCompactionType2ThriftType
```
methods, everywhere else `CompactionType` enum is used. Not sure what would be the benefit of introducing one more enum here.

Reply:
would be creat if we could have char codes in `CompactionType` enum

Reply:
CompacionType is thrift generated, unfortunately there's no way of adding enum fields there.